rand-color.js

```diff
@@ -0,0 +1,13 @@
+function randColor(array) {
```

Comment:
Pretty cool! I like that you created this yourself without looking for an add-on.